sources/Application/Views/InstrumentView.cpp

```diff
@@ -173,6 +173,11 @@ void InstrumentView::fillSampleParameters() {
   v = instrument->FindVariable(SIP_LOOPMODE);
   f1 = new UIIntVarField(position, *v, "loop mode: %s", 0, SILM_LAST - 1, 1, 1);
   T_SimpleList<UIField>::Insert(f1);
+  position._y += 1;
```

Comment:
Is this adjusted for picoTracker? I reduced the screen size in picoTracker so postions for options would have to be adjusted (and there is no extra space, we'll need to check what's the best way to fit this)

Reply:
Ah gotcha, I didn't consider the screen size
the ideal way of doing it would be to hide slices when not in slice mode and show number of slices to the right when in slice mode. Couldn't figure out how to do this without changing screens though.

Reply:
I'll just chime in to say that screen space is at a bit of a premium on the Instrument screen and I also wanted to add ADSR controls too, so perhaps after this PR is merged we could re-arrange the screen so I've made a [separate issue for that](https://github.com/democloid/picoTracker/issues/62).